### PR TITLE
fix(CI): publish demos to another repo

### PR DESCRIPTION
**Pull request by benjamincharity**
- Now building demo app with a custom base href
- Pushing demos to custom repo

---

Once this is merged in, I will a) update all readme links, & b) back-port to old versions.

### CODE REVIEW

File: tooling/ci/demos-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Deploy demos to S3
+#
+#
+
+. ~/.bashrc
+
+echo 'Current Branch: ' $CIRCLE_BRANCH
+
+# Docs and demos will only deploy when merging to one of these branches
+# TODO: REMOVE BRANCH
```
Comment (shani-terminus) [inline tooling/ci/demos-deploy.sh]: these comments aren't needed

File: .circleci/config.yml [hunk 2]
```diff
@@ -30,16 +30,16 @@ jobs:
...
-          name: Lint
-          command: yarn run lint:ci
+      #- run:
+          #name: Lint
+          #command: yarn run lint:ci
 
-      - run:
-          name: Run Tests
-          command: tooling/ci/test-unit.sh
+      #- run:
```
Comment (shani-terminus) [inline .circleci/config.yml]: are these commented out so we can reference them later?

File: tooling/ci/s3-deploy.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@
 echo 'Current Branch: ' $CIRCLE_BRANCH
 
 # Docs and demos will only deploy when merging to one of these branches
-releaseBranches=("master" "8.x.x" "9.x.x" "10.x.x")
+releaseBranches=("master" "8.x.x" "9.x.x" "10.x.x" "11.x.x")
```
Comment (shani-terminus) [inline tooling/ci/s3-deploy.sh]: "1112-demos-deep-linking" ?

File: tooling/ci/demos-deploy.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# TODO: REMOVE BRANCH
+releaseBranches=("master" "8.x.x" "9.x.x" "10.x.x" "11.x.x" "1112-demos-deep-linking")
+
+
+# Only publish demos if this is a release branch
+if [[ " ${releaseBranches[@]} " =~ " ${CIRCLE_BRANCH} " ]]; then
+  # Push changes
+  git config credential.helper 'cache --timeout=120'
+  git config user.email "ben@benjamincharity.com"
```
Comment (shani-terminus) [inline tooling/ci/demos-deploy.sh]: do you want to expose your user info?

Comment (benjamincharity) [reply tooling/ci/demos-deploy.sh]: It's only already public info

**Pull request merged**